README.md

```diff
@@ -1,44 +1,60 @@
-# YTerMusic
+<h1><a id="YTerMusic" class="anchor" href="#YTerMusic"></a></h1>
 
-![index](https://user-images.githubusercontent.com/16625987/202790566-9b114019-63f9-4c4b-965d-820fd0d80a17.png)
+![YTeRMUSiC](./assets/banner/YTeRMUSiC.png)
 
-YTerMusic is a terminal based Youtube Music Player.
-It's aims to be as fast and simple as possible.
+YTerMusic is a TUI based Youtube Music Player that aims to be as fast and simple as possible.
 
-## Features
+## Screenshots
+<p>
+<img src="./assets/screenshots/Choose-a-playlist.png" alt="Choose-a-playlist"/>
+<img src="./assets/screenshots/Playlist-Green.png" alt="Playlist-Green" width="49%"/>
+<img src="./assets/screenshots/Playlist-Blue.png" alt="Playlist-Blue" width="49%"/>
+<img src="./assets/screenshots/Playlist-Red.png" alt="Playlist-Red" width="49%"/>
+<img src="./assets/screenshots/Playlist-without-volume-slider.png" alt="Playlist-without-volume-slider" width="49%"/>
+</p>
+
+## Features and upcoming features
 
 - Play your Youtube Music Playlist and Supermix.
 - Memory efficient (Around 20MB of RAM while fully loaded)
 - Cache all downloads and store them
 - Work even without connection (If musics were already downloaded)
 - Automatic background download manager
+	### Check List 
+	- [x] Playlist selector
+	- [x] Error message display in the TUI
+	- [x] Enable connection less music playing
+	- [ ] Cache limit to not exceed some given disk space
+	- [x] A download limit to stop downloading after the queue is full
+	- [x] Mouse support
+	- [x] Search
+	- [ ] Custom theming
+
+## Install
+- Download the latest version from [releases](https://github.com/ccgauche/ytermusic/releases/latest)
+	### Linux
+	> 3rd party AUR packages are available [here](https://aur.archlinux.org/packages?O=0&K=ytermusic).
+	
+	Install the following libraries:
+	
+	```
```

Comment:
```suggestion
	```sh
```

Reply:
That really doesn't make any difference. If it at least added syntax highlighting I'd might have done that, but whether or not its there is inconsequential.

Reply:
@ccgauche don't pull is quite yet. there's a small change I want to do.